frontend/src/main.tsx

```diff
@@ -25,24 +26,32 @@ import '@/v4/services/fontAwesome';
 import 'simplebar/dist/simplebar.min.css';
 
 import 'simplebar';
```

Comment:
I checked that this is for custom scrollbars, but we should use `simplebar-react`. Currently, both packages are installed, but in fact, neither is used. Do we want to leave it like that?

Reply:
@sanmont3drepo ignore this comment, I realize we are using one webpack configuration and one output file. 
But this can cause us problems in the future. Did you consider moving the new version of the application completely to the subfolder, along with the configuration of the webpack, package.json etc.?
I think this would be much safer. Inadvertently, it will be easy to break something in the old version of the application and not even notice it.